### ISSUE TITLE
Full path in js files

### ISSUE THREAD

**Issue by wonkybream** [python-discord/django-simple-bulma#9]
I have following problems with JS files loaded into template.  

Your loader uses full path which in my case means that path contains site-packages and all that stuff  
`/lib/python3.6/site-packages/django_simple_bulma/js/bulma-tagsinput.js`

Loading `css` works fine because it's done like this `css = static("css/bulma.css")` instead of using full path.

By modifying following code to use `filename.name` to construct path seems to work in my case.
``` python
@register.simple_tag
def bulma():
    """
    Build and return all the HTML required to
    import bulma and the javascript for all
    active extensions.
    """
    # Build the html to include the stylesheet
    css = static("css/bulma.css")
    html = [f'<link rel="stylesheet" href="{css}">']

    # Build html to include all the js files required.
    for filename in js_folder.iterdir():
        js_file = static(f"js/{filename.name}")
        extension_name = filename.stem

        if extension_name in extensions or extensions == "_all":
            html.append(f'{" " * 8}<script type="text/javascript" src="{js_file}"></script>')

    return mark_safe("\n".join(html))  # noqa
```

Now to my actual question.
Is there something wrong on my end or is there actually small bug in code?

Thank you very much for your work with this and if there's anything I can do to help further just say a word.

**Comment by gdude2002** [comment]
Hey, thanks for your ticket.

Everyone's super busy right now with Christmas and other winter festivities, but I'll assign your ticket and we'll take a look then we can.

**Comment by heavysaturn** [comment]
Yeah @wonkybream, I had a quick look now and it looks like you're absolutely right about this being an oversight on my end. it should definitely be `filename.name`. I'm actually surprised I got it to work at all with the full file path being used. Perhaps the javascript was never working, I'm not sure it's been tested at all.

If you submit a pull request, I'd be happy to accept it. Otherwise, I can make this change and get a new version pushed to PyPI some day later this week when my house isn't full of visiting family members. :christmas_tree: :santa:

**Comment by wonkybream** [comment]
Glad to help, if I get some free time from all these relatives and eating fests I'll try to do pull requests.

Great holidays and thanks again for great project!

**Comment by heavysaturn** [comment]
Solved in 1.0.4.
